### DISCUSSION TITLE
How to make q-scrollarea "autogrow" to some size, than scroll?

### DISCUSSION THREAD

**Discussion by FragsterAt** [quasarframework/quasar#15559] [General - Components / Directives / etc] [answered]
How to make similar max-height behavior?
https://codepen.io/FragsterAt/pen/rNZdzNy

**Comment by pdanpdan** [comment]
https://codepen.io/pdanpdan/pen/KKxoXKy

**Reply by FragsterAt** [reply]
@pdanpdan , no that's not it. .scrollarea-container has a fixed height. 
I have updated https://codepen.io/FragsterAt/pen/rNZdzNy , add comments

**Reply by pdanpdan** [reply]
Move styles on parent and account for padding and border
updated codepen

**Reply by FragsterAt** [reply]
> Move styles on parent and account for padding and border updated codepen

basically it works. 
Is there solution without js?

**Comment by pdanpdan** [comment, marked as answer]
Just by duplicating the content and placing the scroll area over it positioned absolute
